features/manage-position/Create.tsx

```diff
@@ -194,68 +191,76 @@ const Create = () => {
             them if you want to short the underlying.
           </Typography>
         </Box>
-        <Box py={2}>
+        <Box pb={2}>
           <Typography>
-            When you're ready, fill in the desired amount of collateral and
-            tokens below and click the "Create" button.
+            When you're ready, fill in the amount of collateral and tokens below
+            and click Create.
           </Typography>
         </Box>
       </Box>
 
-      <Box py={2}>
-        <TextField
-          type="number"
-          label={`Collateral (${collSymbol})`}
-          placeholder="1234"
-          inputProps={{ min: "0" }}
-          value={collateral}
-          error={balanceTooLow}
-          helperText={balanceTooLow ? "Balance too low" : null}
-          onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
-            setCollateral(e.target.value)
-          }
-        />
-      </Box>
-      <Box py={2}>
-        <TextField
-          type="number"
-          label={`Tokens (${tokenSymbol})`}
-          placeholder="1234"
-          inputProps={{ min: "0" }}
-          value={tokens}
-          onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
-            setTokens(e.target.value)
-          }
-        />
-      </Box>
-      <Box py={2}>
-        {needAllowance() && (
-          <Button
-            variant="contained"
-            onClick={setMaxAllowance}
-            style={{ marginRight: `12px` }}
-          >
-            Approve
-          </Button>
-        )}
-        {tokens &&
-        collateral &&
-        gcr &&
-        !needAllowance() &&
-        computedCR > gcr &&
-        !balanceTooLow ? (
-          <Button
-            variant="contained"
-            onClick={handleCreateClick}
-          >{`Create ${tokens} ${tokenSymbol} with ${collateral} ${collSymbol}`}</Button>
-        ) : (
-          <Button variant="contained" disabled>
-            Create
-          </Button>
-        )}
-      </Box>
+      <Grid container spacing={3}>
```

Comment:
I'm going to ignore the fact that this renders horribly on Mobile, but I assume the entire site currently does anyways. 
![Screen Shot 2020-07-23 at 20 32 28](https://user-images.githubusercontent.com/9457025/88351249-a2607000-cd23-11ea-9334-38325b8fccca.png)

Reply:
ye, the whole dapp looks trash on mobile. I will fix this up in a follow up PR. issue #41 is tracking this.